src/digest/base.cr

```diff
@@ -0,0 +1,83 @@
+require "value_ref"
+require "base64"
+
+module Digest::Base(C)
```

Comment:
Convention. Reading `C` made we wondered wth it was, and why someone would call a struct/class like that, until I noticed it denoted a generic. Reading `T` I immediately understand it as a generic type.